FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/physical/Robot.java

```diff
@@ -69,25 +74,25 @@
 	@Override
     public void robotInit() {
     	SmartDashboard.putNumber("LIDAR Speed", 80);		
-//    	SmartDashboard.putNumber("Motor Tweak", MOTOR_POWER_FACTOR);
+    	// SmartDashboard.putNumber("Motor Tweak", MOTOR_POWER_FACTOR);
 		
 		// Initialize the physical components before the controllers,
 		// in case they depend on them.
 		// rightSlave.changeControlMode(TalonControlMode.Follower);
 		// rightSlave.set(RIGHT_INDEX);
-		rightSlave.setInverted(true);
-		rightMotor.setInverted(true);
+		leftSlave.setInverted(true);

```

Comment:
Again, this is a situation of what this specific actual robot is like vs the lab robot we were using.